### PR TITLE
Create-integrations-update.md

### PR DESCRIPTION
**Pull request by lcolston08**

### CODE REVIEW

File: content/blogposts/2022/integrations-update.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: "A new model for Sourcegraph integrations" 
+description: "Sourcegraph is making updates to our integrations and extensions with the upcoming September release of Sourcegraph 4.0." 
+  - name: Ryan Scott
```
Comment (iskyOS) [inline content/blogposts/2022/integrations-update.md:4]:
```suggestion
authors:
    - name: Ryan Scott
```

File: content/blogposts/2022/integrations-update.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title: "A new model for Sourcegraph integrations" 
+description: "Sourcegraph is making updates to our integrations and extensions with the upcoming September release of Sourcegraph 4.0." 
+authors:
+    - name: Ryan Scott
+    url:
```
Comment (iskyOS) [inline content/blogposts/2022/integrations-update.md:6]: This line may just need to be deleted

```suggestion
```

File: content/blogposts/2022/integrations-update.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/integrations-update/integrations-update-hero.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/integrations-update/integrations-update-social.png
+published: true
+---
+
+With the upcoming launch of Sourcegraph 4.0 in September, 2022, we’re investing in a new model for integrations that will support deeper integrations with our code intelligence platform and make it easier to surface code context during the ideal moments in a developer’s workflow. 
+
+This decision means that after the September 2022 release of Sourcegraph 4.0 the following changes will go into effect: 
+
+* Top used extensions, including code navigation, git-extras, open-in-editor, and search-exports, will become part of the core functionality. These enhancements will provide developers code context while staying in the flow within your preferred tools.
```
Comment (abschu) [inline content/blogposts/2022/integrations-update.md:18]:
```suggestion
* Top used extensions, including code navigation, git-extras, open-in-editor, and search-exports, will become part of the core functionality. These enhancements will provide developers code context while staying in the flow within their preferred tools. 
```

**Review by abschu — APPROVED** [review]
Approved with one minor edit

File: content/blogposts/2022/integrations-update.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+---
+title: "A new model for Sourcegraph integrations" 
+description: "Sourcegraph is making updates to our integrations and extensions with the upcoming September release of Sourcegraph 4.0." 
+authors:
+    - name: Ryan Scott
+publishDate: 2022-08-16 T18:00+02:00
```
Comment (bretthayes) [inline content/blogposts/2022/integrations-update.md:6]: Looks like there is a syntax error with the date. Just need to fix the space:

```suggestion
publishDate: 2022-08-16T18:00+02:00
```

**Review by abschu — APPROVED** [review]

**Pull request merged**